packages/cli-kit/src/public/node/themes/api.ts

```diff
@@ -342,10 +421,8 @@ async function parseThemeFileContent(body: OnlineStoreThemeFileBody): Promise<st
   switch (body.__typename) {
     case 'OnlineStoreThemeFileBodyText':
       return body.content
-      break
     case 'OnlineStoreThemeFileBodyBase64':
       return Buffer.from(body.contentBase64, 'base64').toString()
-      break
```

Comment:
How do these changes play into it?

Reply:
Just cleanup. There's no point in having a break statement after a return?